episodes/13-synchronization.md

```diff
@@ -235,16 +229,14 @@ coforall id in 1..numtasks
     lock.waitFor(numtasks);     //then it waits for lock to be equal numtasks (which will happen when all tasks say hello)
     writeln("task ",id," is done...");
 }
-~~~
-{: .source}
+```
 
-~~~
+```bash
 >> chpl atomic_example.chpl -o atomic_example
 >> ./atomic_example
-~~~
-{: .bash}
+```
 
-~~~
+```output
 greetings form task 4... I am waiting for all tasks to say hello
```

Comment:
Please fix the "form" typo in the code and output.

Reply:
Resolved through a commit.